### PR TITLE
fix: make adding image or file resize correctly

### PR DESCRIPTION
**Pull request by leangseu-edx**
**TL;DR -** send an event to resize on iframe once the image loaded

This code can't be test locally. But I tested on it on Sandbox.

JIRA: https://2u-internal.atlassian.net/browse/AU-1155

**What changed?**

- send resize to the iframe when the image loaded
- see `oa_response.js` file only

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [x] Bumped version number in [setup.py](https://github.com/openedx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/openedx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Review by muselesscreator — APPROVED** [review]
The logic seems sound to me, though I also cannot test it

**Review by schenedx — APPROVED** [review]
👍
